### PR TITLE
Welcome blogpost for Taj and Debs

### PR DESCRIPTION
**Pull request by NPDebs**
This PR addresses [issue #33](https://github.com/open-life-science/notes-and-todos/issues/33)  on the to-do list.
The blogpost welcoming new staff, Taj and Debs has been created.
(Link to deployed page [here](https://deploy-preview-611--ols-bebatut.netlify.app/posts/2023/05/05/welcoming-new-ols-staff)...)

ols-staff-post

**Note for reviewers:** 
1. Taj is technically an author too, but he is not on the OLS site (yet).
2. Ideas on a background image (to use in the hero section), please? 🙏

Thank you.

### CODE REVIEW

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Welcoming new OLS Staff — Debs and Tajuddeen"
+authors:
+- yochannah
+- npdebs
+---
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:7]: For the hero image, maybe something with keywords "welcome" or "welcoming party".
Maybe: https://www.pexels.com/photo/person-standing-near-brown-welcome-on-board-printed-floor-map-128299/

Other location to look for
- Unsplash: https://unsplash.com/s/photos/welcoming-party
- Pexels: https://www.pexels.com/search/welcome/

Few criteria
- Better to have something a bit neutral
- Better to have landscape image

**Review by bebatut — COMMENTED** [review]
Thanks @NPDebs for the post.
I made some comments directly in the post. Another one: would it be possible to highlight some keywords in the text? It helps to read the post. Thanks a lot

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+_This blog is in [Nigerian Pidgin](https://en.wikipedia.org/wiki/Nigerian_Pidgin) and in [English](https://en.wikipedia.org/wiki/English_language). Scroll down to find the English-language translation._
+
+## Taj
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply _posts/2023-05-05-welcoming-new-ols-staff.md:11]: Sounds good to me. I'll check in with Taj and see if he is okay with it.

File: _posts/2023-05-05-welcoming-new-ols-staff.md (hunk 1 above)
Comment (NPDebs) [reply _posts/2023-05-05-welcoming-new-ols-staff.md:7]: This image is perfect! And I know where and how to pick images from next time. Thanks a lot!

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Welcoming new OLS Staff — Debs and Tajuddeen"
+authors:
+- yochannah
+- npdebs
+---
+
+_This blog is in [Nigerian Pidgin](https://en.wikipedia.org/wiki/Nigerian_Pidgin) and in [English](https://en.wikipedia.org/wiki/English_language). Scroll down to find the English-language translation._
```
[the remark this answers is not in the document]
Comment (NPDebs) [reply _posts/2023-05-05-welcoming-new-ols-staff.md:13]: Done. Thank you!

File: _posts/2023-05-05-welcoming-new-ols-staff.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (NPDebs) [reply _posts/2023-05-05-welcoming-new-ols-staff.md]: Done. I'll add a photo for Taj (in the morning) immediately he sends it in.

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+The things I no fit do for dark chocolate and ice cream no too plenty.
+  </div>
+</div>
+
+## English
+
+We're delighted to share that OLS has two new staff members: Tajuddeen, who has joined OLS as the coordinator for our [collaborative ECB grant]({% link _posts/2022-12-20-ECB-grant-announcement.md %}), and Debs, who has joined the OLS team as a junior web developer. Between Tajuddeen, Debs, and Jilaga (OLS's Outreachy intern for re-branding), Nigerians are now the most highly-represented nationality in the OLS team! 
+
+
+### More about Taj, Programme Manager:
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:51]:
```suggestion
### Taj, Programme Manager
```

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+### More about Taj, Programme Manager: 
+
+My name is Tajuddeen Gwadabe and I have a PhD in Computer Science and Technology from the University of Chinese Academy of Science. I am really interested in organising and coordinating stuff and that led me to a career in project management. 
+
+I passionately care about learning new cultures and meeting people from all over the world. Currently I have interacted with people from other 40 countries either physically or virtually. 
+
+I watch a lot of videos on sustainability in agriculture, climate, building and technology.
+
+
+### More about Debs, Junior Web Developer:
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:60]:
```suggestion
### Debs, Junior Web Developer
```

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Welcoming new OLS Staff — Debs and Tajuddeen"
+authors:
+- yochannah
+- npdebs
+image: https://images.pexels.com/photos/128299/pexels-photo-128299.jpeg
+photos:
+  name: Mabel Amber
+  url: https://www.pexels.com/photo/person-standing-near-brown-welcome-on-board-printed-floor-map-128299/
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:10]:
```suggestion
  url: https://www.pexels.com/photo/person-standing-near-brown-welcome-on-board-printed-floor-map-128299/
  licence: CC-BY
```

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+_This blog is in [Nigerian Pidgin](https://en.wikipedia.org/wiki/Nigerian_Pidgin) and in [English](https://en.wikipedia.org/wiki/English_language). Scroll down to find the English-language translation._
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:13]:
```suggestion
We’re delighted to share that OLS has two new staff members: Tajuddeen, who has joined OLS as the coordinator for our [collaborative ECB grant](https://deploy-preview-611--ols-bebatut.netlify.app/posts/2022/12/20/ECB-grant-announcement/), and Debs, who has joined the OLS team as a junior web developer. Between Tajuddeen, Debs, and Jilaga (OLS’s Outreachy intern for re-branding), Nigerians are now the most highly-represented nationality in the OLS team!

_This blog is in [Nigerian Pidgin](https://en.wikipedia.org/wiki/Nigerian_Pidgin) and in [English](https://en.wikipedia.org/wiki/English_language). Scroll down to find the English-language translation._
```

File: _posts/2023-05-05-welcoming-new-ols-staff.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+I too like food, podcast, photography, horror feem and to tell computer wetin to do (* winch laugh *)
+
+The things I no fit do for dark chocolate and ice cream no too plenty.
+  </div>
+</div>
+
+## English
+
+We're delighted to share that OLS has two new staff members: Tajuddeen, who has joined OLS as the coordinator for our [collaborative ECB grant]({% link _posts/2022-12-20-ECB-grant-announcement.md %}), and Debs, who has joined the OLS team as a junior web developer. Between Tajuddeen, Debs, and Jilaga (OLS's Outreachy intern for re-branding), Nigerians are now the most highly-represented nationality in the OLS team!
```
Comment (bebatut) [inline _posts/2023-05-05-welcoming-new-ols-staff.md:48]:
```suggestion
```

**Comment by bebatut** [comment]
Looks good now. Thanks @NPDebs

**Pull request merged**